src/main/java/Jane.java

```diff
@@ -0,0 +1,269 @@
+            }
+        } catch (IOException err) {
+            System.out.println("Unable to create directory");
+            err.printStackTrace();
+        //Check if the janelist exist inside the folder we have made and if it doesnt, create, if not read it
+        } try {
+            if (Files.notExists(filePath)) {
+                Files.createFile(filePath);
+            } else {
+                List<String> lines = Files.readAllLines(filePath);
+                for (String s:lines) {
+                    //to separate each portion of the task eg D | taskname | deadline to easily see which type of task and deadline
+                    String[] line = s.split("\\|");
+                    int i = Integer.parseInt(line[1]);
+                    boolean b = (i==1);
+                    if (line[0].equals("T")) {
+                        Todo T = new Todo(tasks.size() + 1, line[2]);
+                        T.changeState(b);
+                        tasks.add(T);
+                    } else if (line[0].equals("D")) {
+                        Deadline D = new Deadline(tasks.size() + 1, line[2], LocalDateTime.parse(line[3]));
+                        D.changeState(b);
+                        tasks.add(D);
+                    } else if (line[0].equals("E")) {
+                        Event E = new Event(tasks.size()+1, line[2], LocalDateTime.parse(line[3]), LocalDateTime.parse(line[4]));
+                        E.changeState(b);
+                        tasks.add(E);
+                    }
+
+                }
+
+
+            }
+
+        } catch (IOException err) {
+            System.out.println("Unable to create File");
+            err.printStackTrace();
+        }
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        Scanner in = new Scanner(System.in);
+        int count = 0;
+        while (in.hasNext()) {
+            String output = in.nextLine();
+            if (output.equals("bye")) {
+                List<String> currentList = new ArrayList<>();
+                for (Task t : tasks) {
+                    currentList.add(t.save());
+                }
+                try {
+                    Files.write(filePath, currentList);
+                }
+                catch (IOException err)  {
+                    System.out.println("cannot save list");
+                    err.printStackTrace();
+                }
+                break;
+            }
+            else if(output.startsWith("mark")) {
+                String[] s = output.split(" ");
+                int num = Integer.parseInt(s[1]);
+                if (num < count+1) {
+                    System.out.println("Nice! I've marked this task as done");
+                    Task n = tasks.get(num-1);
```

Comment:
[Operators should be surrounded by a space character](https://se-education.org/guides/conventions/java/intermediate.html#layout)